doc/classes/EditorSceneFormatImporter.xml

```diff
@@ -13,9 +13,10 @@
 		<method name="_get_extensions" qualifiers="virtual const">
 			<return type="PackedStringArray" />
 			<description>
+				Return supported file extensions for this scene importer.
 			</description>
 		</method>
-		<method name="_get_import_flags" qualifiers="virtual const">
+		<method name="_get_import_flags" qualifiers="virtual const" deprecated="Appears to be unused">
```

Comment:
Seems like this has been unused since `GODOT IS OPEN SOURCE`... it's about time we notice.
```suggestion
		<method name="_get_import_flags" qualifiers="virtual const" deprecated="Unused by the engine, and will be removed in a future version. Implementing this has no effect.">
```

And I'd suggest enclosing the code related to `_get_import_flags` and `get_import_flags` in `#ifndef DISABLE_DEPRECATED`, so we find it and nuke it next time we decide to remove deprecated methods.

Reply:
See 75d5242, I think we can live with that minor compat breakage.

Reply:
Done with #101531.